src/libstd/panicking.rs

```diff
@@ -208,9 +207,8 @@ pub fn on_panic(obj: &(Any+Send), file: &'static str, line: u32) {
     // abort immediately to avoid infinite recursion, so that attaching a
     // debugger provides a useable stacktrace.
     if panics >= 3 {
-        util::dumb_print(format_args!("thread panicked while processing \
-                                       panic. aborting.\n"));
-        unsafe { intrinsics::abort() }
+        rtabort!("thread panicked while processing \
+                  panic. aborting.");
```

Comment:
This means that these messages will now be prefixed with `fatal runtime error:` rather than being printed directly as-is. In the past I think we tried to avoid printing the "fatal runtime error" aspect?